src/test/java/org/dice_research/opal/launuts/polygons/parser/NutParserTest.java

```diff
@@ -120,7 +120,7 @@ public void testCaseForHoleTestingOfWeimarLand()
 	}
 
 	@Test
-	public void testCaseForHoleTestingOfSüdlicheweinstraße()
+	public void testCaseForHoleTestingOfSuedlicheweinstrasse()
```

Comment:
Please send the pull request to "feature-polygons" branch of "https://github.com/G0URAB/LauNuts".